js/viz.js

```diff
@@ -325,13 +351,11 @@ const viz = {
 
   dragEnd() {
     d3.event.subject.shadow = false;
-    this.endEvent();
```

Comment:
The current lightbeam stops moving after a while so doesn't keep repelling and rendering again.